examples/connector/python/requirements.txt

```diff
@@ -0,0 +1,2 @@
+grpcio==1.60.1
```

Comment:
We should match the versions we are showing in the [development guide](https://github.com/fivetran/fivetran_sdk/blob/main/development-guide.md#versions). Can we not use `1.61.1`?

Reply:
I think there was no `1.61.1 `
<img width="1635" alt="Screenshot 2024-04-15 at 10 37 25 AM" src="https://github.com/fivetran/fivetran_sdk/assets/9125374/f5f4eb14-2580-46aa-b8b3-37036d046f8c">